platform.md

```diff
@@ -276,10 +275,79 @@ All lifecycle phases:
 - MUST read `CNB_PLATFORM_API` from the execution environment and evaluate compatibility before attempting to parse other inputs (see [Platform API Compatibility](#platform-api-compatibility))
 - MUST give command line inputs precedence over other inputs
 
+#### `analyzer`
+The platform MUST execute `analyzer` in the **build environment**
+
+Usage:
+```
+/cnb/lifecycle/analyzer \
+  [-analyzed <analyzed>] \
+  [-daemon] \ # sets <daemon>
+  [-gid <gid>] \
+  [-layers <layers>] \
+  [-log-level <log-level>] \
+  [-order <order>] \
+  [-previous-image <previous-image> ] \
+  [-run-image <run-image> ] \
+  [-stack <stack> ] \
+  [-tag <tag>...] \
+  [-uid <uid>] \
+  <image>
+```
+
+##### Inputs
+| Input             | Environment Variable  | Default Value            | Description
+|-------------------|-----------------------|--------------------------|----------------------
+| `<analyzed>`      | `CNB_ANALYZED_PATH`   | `<layers>/analyzed.toml` | Path to output analysis metadata (see [`analyzed.toml`](#analyzedtoml-toml)
+| `<daemon>`        | `CNB_USE_DAEMON`      | `false`                  | Analyze image from docker daemon
+| `<gid>`           | `CNB_GROUP_ID`        |                          | Primary GID of the stack `User`
+| `<layers>`        | `CNB_LAYERS_DIR`      | `/layers`                | Path to layers directory
+| `<image>`         |                       |                          | Tag reference to which the app image will be written
+| `<log-level>`     | `CNB_LOG_LEVEL`       | `info`                   | Log Level
+| `<order>`         | `CNB_ORDER_PATH`      | `/cnb/order.toml`        | Path to order definition (see [`order.toml`](#ordertoml-toml))
+| `<previous-image>`| `CNB_PREVIOUS_IMAGE`  | `<image>`                | Image reference to be analyzed (usually the result of the previous build)
+| `<run-image>`     | `CNB_RUN_IMAGE`       | resolved from <stack>    | Run image reference
+| `<stack>`         | `CNB_STACK_PATH`      | `/cnb/stack.toml`        | Path to stack file (see [`stack.toml`](#stacktoml-toml))
+| `<tag>...`        |                       |                          | Additional tag to apply to exported image
+| `<uid>`           | `CNB_USER_ID`         |                          | UID of the stack `User`
+
+- **If** `<daemon>` is `false`, `<image>` MUST be a valid image reference
+- **If** `<daemon>` is `true`, `<image>` MUST be either a valid image reference or an imageID
+- **If** the platform provides one or more `<tag>` inputs, each `<tag>` MUST be a valid image reference.
```

Comment:
```suggestion
-`<image>` MUST be a valid image reference
- **If** the platform provides one or more `<tag>` inputs, each `<tag>` MUST be a valid image reference.
- **If** `<daemon>` is `false`, and `<previous-image>`, if provided,  MUST be a valid image reference
- **If** `<daemon>` is `true`, `<previous-image>`, if provided, MUST be either a valid image reference or an imageID
```
Sorry! I screwed this up the first time. `<image>` can't be an imageID in the daemon case, b/c we won't be writing to an imageID, this restriction should instead apply to the `<previous-image>`.